src/libmongoc/tests/test-mongoc-client-side-encryption.c

```diff
@@ -2674,14 +2674,28 @@ test_bypass_spawning_via_helper (const char *auto_encryption_opt)
    } else if (0 == strcmp (auto_encryption_opt, "bypass_query_analysis")) {
       mongoc_auto_encryption_opts_set_bypass_query_analysis (
          auto_encryption_opts, true);
-   } else if (0 == strcmp (auto_encryption_opt, "cryptSharedLibRequired")) {
+   } else if (0 == strcmp (auto_encryption_opt, "crypt_shared_lib_loaded")) {
```

Comment:
```suggestion
   } else if (0 == strcmp (auto_encryption_opt, "cryptSharedLibRequired")) {
```

Suggest keeping original casing. This matches the option value in `mongoc_auto_encryption_opts_set_extra`.